api/features/models.py

```diff
@@ -984,3 +993,52 @@ def get_update_log_message(self, history_instance) -> typing.Optional[str]:
 
     def _get_environment(self) -> typing.Optional["Environment"]:
         return self.feature_state.environment
+
+
+class FeatureExport(models.Model):
+    """
+    Stores the representation of an environment's export of
+    features between the request for the export and the
+    ultimate download. Records are deleted automatically after
+    a waiting period.
+    """
+
+    # The environment the export came from.
+    environment = models.ForeignKey(
+        "environments.Environment",
+        related_name="feature_imports",
+        on_delete=models.CASCADE,
+        swappable=True,
+    )
+
+    # This is a JSON string of data used for file download
+    # once the task has completed assembly.
+    data = models.CharField(max_length=MAX_FEATURE_EXPORT_SIZE)
+    created_date = models.DateTimeField("DateCreated", auto_now_add=True)
+
+
+class FeatureImport(models.Model):
```

Comment:
I thought about adding a field to import to mark whether or not the import was successful but I was on the fence. Thoughts?

Reply:
I think that would be useful, yes. I wonder if we could capture any logs if it's not successful and store them here (see `task_processor.TaskRun` for example).

Reply:
Are we thinking about exposing information to the user that imports the task? Or if we're just storing the the traceback information for our own information is there additional advantage to duplicating it out of the existing task?

Reply:
Talking offline, go with a status field but leave the validation stuff off for now, we can always add it later.